joomla/joomla-cms
# [4.0] com_menus javascript clean up

Pull Request for Issue # .

### Summary of Changes

- move inline code to files
- Drop some jQuery (modals using jQuery, because why not :( )

### Testing Instructions
- Create a new MENU
- Create all the possible menu items
- Check that modules modals work correctly for Menu and Items

### Expected result



### Actual result



### Documentation Changes Required
None code clean up, preparing joomla for strict CSP mode
